charts/backstage/README.md

```diff
@@ -48,8 +48,8 @@ This chart bootstraps a [Backstage](https://backstage.io/docs/deployment/docker)
 
 ## Prerequisites
 
-- Kubernetes 1.19+
-- Helm 3.2.0+
+- Kubernetes 1.25+ (OpenShift 4.12+)
```

Comment:
Does the kubeVersion at https://github.com/redhat-developer/rhdh-chart/pull/8/files#diff-8316e61c38ae132c31764bc73c703fefb6bfe57770e77aa97e63c0f9f490a1fcR32 need to be updated alongside of this?

Reply:
honestly, I only bumped the version to align with our operator minimum version... but yeah, might as well be consistent. Good catch!